src/test/resources/application-test.yml

```diff
@@ -1,6 +1,50 @@
+spring:
+
+  config:
+    use-legacy-processing: true
+
+  profiles:
+    include: stdout
+
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          public-key-location: classpath:local-public-key.pub
+
+  datasource:
+    url: 'jdbc:h2:mem:licences-db;MODE=PostgreSQL'
+    username: create_vary
+    password: dummy
+
+  flyway:
+    locations: classpath:/migration/h2,classpath:/migration/common
+    url: ${spring.datasource.url}
+    user: create_vary
+    password: dummy
+
+  h2:
+    console:
+      enabled: true
+
+  jpa:
+    show-sql: true
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.H2Dialect
+
+  sql:
+    init:
+      continue-on-error: true
+
```

Comment:
Test properties for the resource server, database, flyway and JPA. It uses H2 in test, not postgresql.